src/libraries/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -8,7 +8,7 @@ internal static partial class Libraries
         // Shims
         internal const string SystemNative = "libSystem.Native";
         internal const string NetSecurityNative = "libSystem.Net.Security.Native";
-        internal const string CryptoNative = "libSystem.Security.Cryptography.Native.OpenSsl";
+        internal const string CryptoNative = "libSystem.Security.Cryptography.Native.Android";
```

Comment:
I'm pretty sure this is used on non-Android platforms.

Reply:
There are different possible ways to deal with this:
- Split the Interop.Libraries.cs in a way that on Android builds you get a different name. 
- Add a different name in https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/Interop/Android/Interop.Libraries.cs and then update the references in the Interop files to use it (eg.
https://github.com/dotnet/runtime/blob/66d26c36991c7b5ff09c7af07cae9c6d59510b7c/src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.Bignum.cs#L13)